python/lsst/meas/algorithms/subtractBackground.py

```diff
@@ -320,9 +326,16 @@ def fitBackground(self, maskedImage, nx=0, ny=0, algorithm=None):
         if algorithm is None:
             algorithm = self.config.algorithm
 
-        bctrl = afwMath.BackgroundControl(algorithm, nx, ny,
-                                          self.config.undersampleStyle, sctrl,
-                                          self.config.statisticsProperty)
+        # This call to a deprecated BackgroundControl constructor is necessary
+        # to support the algorithm parameter; it should be replaced with
+        #
+        #     afwMath.BackgroundControl(nx, ny, sctrl, self.config.statisticsProperty)
+        #
+        # when algorithm has been deprecated and removed (DM-22814).
```

Comment:
Should this be marked TODO or is that optional when the ticket is mentioned?

Reply:
I'm not sure that we actually have a rule about that, but I think it would be a good idea to do so. I'll change it.